recipes/pysdl2/locate_sdl.patch

```diff
@@ -0,0 +1,18 @@
+--- sdl2/__init__.py
++++ sdl2/__init__.py
+@@ -1,4 +1,15 @@
+ """SDL2 wrapper package"""
++# Direct this module to Ana/Miniconda's lib folder, where the SDL2 libraries
++# can be found.
++import os
++
++if os.name == "posix":
++	os.environ["PYSDL2_DLL_PATH"] = os.path.abspath(os.path.join(
++		os.path.dirname(__file__), os.pardir, os.pardir, os.pardir))
++elif os.name == "nt":
++	os.environ["PYSDL2_DLL_PATH"] = os.path.abspath(os.path.join(
++		os.path.dirname(__file__), os.pardir, os.pardir, os.pardir, "Library", "bin"))
```

Comment:
So I have reinstated the original patch, with the change that it checks if the PYSDL2_DLL_PATH variable already exists, and only sets it to the conda dirs if it doesn't. This still maybe isn't ideal, but is it good enough for the moment to accept these recipes? 
I am willing to create an issue report upstream at pysdl2 so it might be fixed in a better manner, but can you help me a bit in what I exactly should ask? How do you obtain information about the prefix? It is not an environment variable isn't it? I can find the environment variable CONDA_PREFIX, but this is only created when running from a virtual environment, not in the main conda environment.